### PR TITLE
feat: add v10 and promote it to latest

### PR DESCRIPTION
**Pull request by Duologic**
Celebrating the :100: :star:, this PR adds v10 and promotes it to the latest.

### CODE REVIEW

File: grafonnet-base/main.libsonnet
Comment (Duologic) [inline grafonnet-base/main.libsonnet:1]: The actual changes to support this happen in this file.

File: grafonnet-base/main.libsonnet [hunk 1]
```diff
@@ -187,9 +194,14 @@ local veneer = import './veneer/main.libsonnet';
   coreLib: {
     new(schema):
       local title = schema.info.title;
+      local spec =
+        if 'spec' in schema.components.schemas[title].properties
+        then schema.components.schemas[title].properties.spec
+        else schema.components.schemas[title];
```
Comment (Duologic) [inline grafonnet-base/main.libsonnet:200]: The new schemas render the core kinds as Kubernetes-like resources, this little check ensures we use the spec of those resources.

File: docs/API/panel/heatmap/index.md [hunk 2]
```diff
@@ -515,9 +515,10 @@ Controls frame rows options
 withShowValue(value)
 ```
 
-TODO docs
-
-Accepted values for `value` are "auto", "never", "always"
+| *{
+	layout: ui.HeatmapCellLayout & "auto" // TODO: fix after remove when https://github.com/grafana/cuetsy/issues/74 is fixed
+}
```
Comment (Duologic) [inline docs/API/panel/heatmap/index.md:520]: Interesting artifact propagating through the docstring, prolly need to prevent that somehow.

**Review by K-Phoen — APPROVED** [review]
:rocket:

**Pull request merged**